appeals/web/src/server/appeals/national-list/national-list.service.js

```diff
@@ -1,16 +1,17 @@
-import { get } from '../../lib/request.js';
 import { paginationDefaultSettings } from '../appeal.constants.js';
 
 /** @typedef {import('@pins/appeals').AppealList} AppealList */
 
 /**
+ * @param {import('got').Got} apiClient
  * @param {number} pageNumber
  * @param {number} pageSize
  * @param {string} searchParam
  * @returns {Promise<AppealList>}
  */
 export const getAppealsByPage = (
+	apiClient,
 	pageNumber = paginationDefaultSettings.firstPageNumber,
 	pageSize = paginationDefaultSettings.pageSize,
 	searchParam = ''
-) => get(`appeals?pageNumber=${pageNumber}&pageSize=${pageSize}${searchParam}`);
+) => apiClient.get(`appeals?pageNumber=${pageNumber}&pageSize=${pageSize}${searchParam}`).json();
```

Comment:
Had to put `.json()` here, as TS doesn't seem to understand the client config (responseType: 'json',
	resolveBodyOnly: true)

Reply:
That'll be because the type used everywhere is `import('got').Got` so it doesn't know it's been configured.

Reply:
how would I solve it? `import('got').Response`?

Reply:
or importing the instance maybe...

Reply:
yeah I can't think of a specific way, as `got.extend` just returns `Got` type.

Reply:
Looks like others have this issue, with no obvious fix: https://github.com/sindresorhus/got/issues/1117